File: app/src/main/java/com/edricchan/studybuddy/NoSwipeBehavior.java
package com.edricchan.studybuddy;

import android.view.View;

import com.google.android.material.snackbar.BaseTransientBottomBar;

public class NoSwipeBehavior extends BaseTransientBottomBar.Behavior {

	@Override
	public boolean canSwipeDismissView(View child) {
		return false;
	}
}

File: app/src/main/java/com/edricchan/studybuddy/NoSwipeBehavior.kt
package com.edricchan.studybuddy

import android.view.View

import com.google.android.material.snackbar.BaseTransientBottomBar

class NoSwipeBehavior : BaseTransientBottomBar.Behavior() {

	override fun canSwipeDismissView(child: View): Boolean {
		return false
	}
}
